.github/workflows/downstream-changelog.yml

```diff
@@ -0,0 +1,70 @@
+name: downstream-changelog
+on:
+    - pull_request:
+          types:
+              - closed
```

Comment:
Is `closed` the right type here? That sounds like this will run even on PRs that we've decided to abandon.

Reply:
Yep, the `if: github.event.pull_request.merged` is what makes it work.